website/docs/d/dms_certificate.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+subcategory: "DMS (Database Migration)"
+layout: "aws"
+page_title: "AWS: aws_dms_certificate"
+description: |-
+  Terraform data source for managing an AWS DMS (Database Migration) Certificate.
+---
+
+# Data Source: aws_dms_certificate
+
+Terraform data source for managing an AWS DMS (Database Migration) Certificate.
+
+## Example Usage
+
+### Basic Usage
+
+```terraform
+data "aws_dms_certificate" "example" {
+  certificate_id = aws_dms_certificate.test.certificate_id
+}
+```
+
+## Argument Reference
+
+The following arguments are required:
+
+* `certificate_id` - (Required) A customer-assigned name for the certificate. Identifiers must begin with a letter and must contain only ASCII letters, digits, and hyphens. They can't end with a hyphen or contain two consecutive hyphens.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `certificate_creation_date` - (Optional) The date that the certificate was created.
+* `certificate_pem` - (Optional) The contents of a .pem file, which contains an X.509 certificate.
+* `certificate_owner` - (Optional) The owner of the certificate.
+* `certificate_arn` - (Optional) The Amazon Resource Name (ARN) for the certificate.
+* `certificate_wallet` - (Optional) The owner of the certificate.
+* `key_length` - (Optional) The key length of the cryptographic algorithm being used.
+* `signing_algorithm` - (Optional) The algorithm for the certificate.
+* `valid_from_date` - (Optional) The beginning date that the certificate is valid.
+* `valid_to_date` - (Optional) The final date that the certificate is valid.
```

Comment:
The `(Optional)` can be dropped in the attribute section where everything is computed.